financial-connections/src/main/java/com/stripe/android/financialconnections/domain/NativeAuthFlowCoordinator.kt

```diff
@@ -22,13 +21,6 @@ internal class NativeAuthFlowCoordinator @Inject constructor() {
     }
 
     internal sealed interface Message {
-        /**
-         * Updates global [FinancialConnectionsAuthorizationSession] instance.
-         */
-        data class UpdateAuthorizationSession(
-            val authorizationSession: FinancialConnectionsAuthorizationSession
-        ) : Message
-
```

Comment:
no need for composables to communicate with activity to update the auth session held in the state. Now repository takes care of it transparently.